frontend/src/pages/WhatIsHealthEquity/WhatIsHealthEquityPage.tsx

```diff
@@ -23,13 +31,22 @@ function WhatIsHealthEquityPage() {
           justify="center"
           alignItems="center"
         >
-          <Grid container item xs={12} sm={12} md={4} className={styles.HeaderImgItem}>
-            <img
-              src="img/pexels-marcus-aurelius-4063919 1.png"
-              className={styles.HeaderImg}
-              alt="A woman in a wheelchair relaxing with a cup of tea"
-            />
-          </Grid>
+          <Hidden smDown>
```

Comment:
Did we decide on whether to hide these images on mobile?  @vanshkumar ?

Reply:
I don't think there was a decision on this, feel free to ping the slack to see what folks think